esp/scm/ws_workunits.ecm

```diff
@@ -1826,9 +1834,48 @@ ESPresponse [exceptions_inline, nil_remove] WUGetNumFileToCopyResponse
     [min_ver("1.69")] int64 Total;
 };
 
+ESPStruct [nil_remove] WUECLQueryActionResult
+{
+    string QueryName;
+    string Action;
+    string Result;
+};
+
+ESPrequest [nil_remove] WUECLQueryActionRequest
+{
+    ESParray<string> QueryNames;
```

Comment:
I think "QueryName" is slightly misleading.  This is really the "Definition" name.  Definition being the proper new term for what we used to call ECL "Attributes".

In fact we may want to allow them to override the default query name so it doesn't have to be the same as the "definition"/file name.

Reply:
Other places we call this the "query main"... meaning the main file / definition for the query.